src/lib/api/services/utils.ts

```diff
@@ -1,10 +1,17 @@
 const packageJSON = require('../../../../package.json');
+import configStore, { CONFIG_KEYS } from '../../Config';
+import { AVAILABLE_ENVS } from '../../Env';
 
 export const getCommonHeaderOptions = () => {
     return {
         headers: {
             'Content-Type': 'application/json',
             'x-fp-cli': `${packageJSON.version}`,
-        }
+        },
     };
-};
\ No newline at end of file
+};
+
+export const getBaseURL = () => {
```

Comment:
why this method is needed ? @manishprajapati52

Reply:
Before we had getBaseURL method in `lib>api>services>url.ts` and was using in one file only. But now, as we are using this method in multiple files, I moved this method to utils.